components/database/postgresql-14/Makefile

```diff
@@ -122,17 +122,18 @@ COMPONENT_TEST_TRANSFORMS= \
 REQUIRED_PACKAGES += library/libxslt
 
```

Comment:
library/libxslt is now automatically detected and can removed here.

Reply:
> library/libxslt is now automatically detected and can removed here.

Yep. Thanks for spotting it.  Fixed.